poly-synth/src/poly-synth-demo.js

```diff
@@ -76,7 +176,7 @@ class PolySynthDemo {
    * @param {Number} value The new gain.
    */
   setGain(value) {
-    this.masterGain_.gain.value = parseFloat(value);
+    this.masterGain_.gain.value = value;
```

Comment:
Why remove the `parseFloat`? Was this a mistake in the previous code that we overlooked?